src/legacy/legacy-grpc-network-client.ts

```diff
@@ -109,39 +115,45 @@ class LegacyGRPCNetworkClient implements LegacyNetworkClient {
     })
   }
 
-  /* eslint-disable class-methods-use-this */
-  public XRPAmount(): XRPAmount {
-    return new XRPAmount()
-  }
-
-  public Payment(): Payment {
-    return new Payment()
-  }
-
-  public Transaction(): Transaction {
-    return new Transaction()
-  }
-
-  public SubmitSignedTransactionRequest(): SubmitSignedTransactionRequest {
-    return new SubmitSignedTransactionRequest()
-  }
-
-  public GetLatestValidatedLedgerSequenceRequest(): GetLatestValidatedLedgerSequenceRequest {
-    return new GetLatestValidatedLedgerSequenceRequest()
-  }
-
-  public GetTransactionStatusRequest(): GetTransactionStatusRequest {
-    return new GetTransactionStatusRequest()
-  }
-
-  public GetAccountInfoRequest(): GetAccountInfoRequest {
-    return new GetAccountInfoRequest()
-  }
-
-  public GetFeeRequest(): GetFeeRequest {
-    return new GetFeeRequest()
+  public async createSignedTransaction(
```

Comment:
What's the rationale for putting this inside of this class? Is it stylistic (you're trying to encapsulate protocol buffers here) or required by types?

Reply:
The idea is to encapsulate all protocol buffers inside of the GRPC network clients to better handle the variation between Node buffers and Web buffers.